### ISSUE TITLE
Client stops responding after play

### ISSUE THREAD

**Issue by Matheuszoly** [HitsparkInteractive/NinOnline#19]
## Problem description

I use the windows 7 home premium operating system (already mentioned by some players that it would give compatibility problems) I managed to fix this in the old client and play normally what I've been doing until today since I met the nin. 

but on the new client it just stops responding after pressing play

## Steps to reproduce issue

1. I updated both the inochi client and the Brazilian client
2. I let them finish loading after closing all open programs on my pc and leaving only the nin launcher
3. when I press play on any of the clients, an error log appears as shown in the print below with the error log...

## Observed result

crash client
## Expected result
open the client to play

## More details

**Affected Client Version:** 
Inochi Client

**First noticed:** 

## Error Logs
![image](https://user-images.githubusercontent.com/88858452/129271326-5244a3b0-235f-4e66-b9fb-020a1f30f5cc.png)

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by RorySoh** [comment]
Can you see if there's a log file in your installation folder?

**Comment by Matheuszoly** [comment]
> Can you see if there's a log file in your installation folder?

Launcher log? yes I see, do you want me to post here one of the attempts to open the client? or the whole file?

**Comment by RorySoh** [comment]
The log file in the data files/logs/ folder

**Comment by Matheuszoly** [comment]
On the client inochi (global) there is no such folder

Looking for "data files/logs/folder" in the Brazilian client I found an error file with the following information:

 7/3/2020 11:41:16 PM
The following error occurred at 'HandleMapNPCTarget' in 'modHandleData'.
Run-time error '9': Subscript out of range.

7/3/2020 11:41:24 PM
The following error occurred at 'HandleMapNPCTarget' in 'modHandleData'.
Run-time error '9': Subscript out of range.

7/3/2020 11:41:37 PM
The following error occurred at 'HandleMapNPCTarget' in 'modHandleData'.
Run-time error '9': Subscript out of range.

7/3/2020 11:41:39 PM
The following error occurred at 'HandleMapNPCTarget' in 'modHandleData'.
Run-time error '9': Subscript out of range.

7/3/2020 11:41:39 PM
The following error occurred at 'HandleMapNPCTarget' in 'modHandleData'.
Run-time error '9': Subscript out of range.

7/3/2020 11:41:39 PM
The following error occurred at 'HandleMapNPCTarget' in 'modHandleData'.
Run-time error '9': Subscript out of range.

7/3/2020 11:41:40 PM
The following error occurred at 'HandleMapNPCTarget' in 'modHandleData'.
Run-time error '9': Subscript out of range.

there is no longer any file or folder about errors I looked in both clients and the error file is from 2020 which is strange


Looking in the Brazilian client for "data files/logs/folder" I found this.
![image](https://user-images.githubusercontent.com/88858452/129281310-0b31ec35-f0c7-446e-94f3-8d1c5a3fd0dd.png)

and as I mentioned in the client inochi (global) I didn't find anything about it.
![image](https://user-images.githubusercontent.com/88858452/129281498-3a2d64b8-1667-442f-b75e-6a689e621c5c.png)



> The log file in the data files/logs/ folder
